types/qunit/test/global-test.ts

```diff
@@ -770,3 +770,7 @@ QUnit.module( "async nested hooks", function( hooks ) {
     assert.ok( true, "async afterEach called" );
   } );
 });
+
+QUnit.onUncaughtException = (error: Error) => {
```

Comment:
This shouldn't be assignable since the declaration says the function accepts `unknown`, right?

Reply:
But you can assign anything to `unknown`.

Reply:
But you're not assigning to `unknown`, you're assigning to `(arg: unknown) => void`. A value of type `(e: Error) => void` isn't assignable to that type, because it requires an `Error` as it's first arg, but `(arg: unknown) => void` can be called with anything—remember function parameter types are contravariant with the type of the function itself, so they work "backwards"

Reply:
You're right, of course (and I was thinking about it backwards), but it was surprising to me that TS doesn't actually complain. TIL that, even when `strictFunctionTypes` is enabled, TS allows bivariance for functions written in method syntax. The reason for this behavior is [noted in the docs](https://www.typescriptlang.org/tsconfig#strictFunctionTypes).

Reply:
I've updated the definition of `QUnit.onUncaughtException` to use function syntax so that parameter contravariance is enforced and updated the test.